tests/prague/eip7692_eof_v1/eip7620_eof_create/test_gas.py

```diff
@@ -0,0 +1,208 @@
+        ),
+        pytest.param(
+            EOFCREATE_GAS,
+            1,
+            True,
+            id="EOFCREATE_with_value_new_acc",
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    ["mem_expansion_size", "mem_expansion_extra_gas"],
+    [
+        pytest.param(0, 0, id="no_mem_expansion"),
+        pytest.param(1, 3, id="1byte_mem_expansion"),
+        pytest.param(32, 3, id="1word_mem_expansion"),
+        pytest.param(33, 6, id="33bytes_mem_expansion"),
+    ],
+)
+@pytest.mark.parametrize(
+    ["initcode", "initcode_execution_cost", "runtime"],
+    [
+        pytest.param(
+            smallest_initcode_subcontainer,
+            smallest_initcode_subcontainer_gas,
+            smallest_runtime_subcontainer,
+            id="smallest_code",
+        ),
+        pytest.param(
+            make_initcode(aborting_container),
+            smallest_initcode_subcontainer_gas,
+            aborting_container,
+            id="aborting_runtime",
+        ),
+        pytest.param(
+            reverting_container, smallest_initcode_subcontainer_gas, None, id="reverting_initcode"
+        ),
+        pytest.param(
+            expensively_reverting_container,
+            expensively_reverting_container_gas,
+            None,
+            id="expensively_reverting_initcode",
+        ),
+        pytest.param(
+            make_initcode(big_runtime_subcontainer),
+            smallest_initcode_subcontainer_gas,
+            big_runtime_subcontainer,
+            id="big_runtime",
+        ),
+        pytest.param(
+            make_initcode(make_factory(smallest_initcode_subcontainer)),
+            smallest_initcode_subcontainer_gas,
+            make_factory(smallest_initcode_subcontainer),
+            id="nested_initcode",
+        ),
+        pytest.param(
+            bigger_initcode_subcontainer,
+            bigger_initcode_subcontainer_gas,
+            smallest_runtime_subcontainer,
+            id="bigger_initcode",
+        ),
+        pytest.param(
+            data_initcode_subcontainer,
+            smallest_initcode_subcontainer_gas,
+            data_runtime_container,
+            id="data_initcode",
+        ),
+        pytest.param(
+            data_appending_initcode_subcontainer,
+            data_appending_initcode_subcontainer_gas,
+            data_runtime_container,
+            id="data_appending_initcode",
+        ),
+    ],
+)
+def test_eofcreate_gas(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    cold_gas: int,
+    value: int,
+    new_account: bool,
+    mem_expansion_size: int,
+    mem_expansion_extra_gas: int,
+    initcode: Container,
+    initcode_execution_cost: int,
+    runtime: Container,
+):
+    """Tests variations of EOFCREATE gas"""
+    initcode_hashing_cost = 6 * ((len(initcode) + 31) // 32)
+    deployed_code_cost = 200 * len(runtime) if runtime else 0
+
+    sender = pre.fund_eoa()
+
+    salt_addresses = [compute_eofcreate_address(sender, i, initcode) for i in range(10)]
```

Comment:
```suggestion
    salt_addresses = [compute_eofcreate_address(sender, i, initcode) for i in range(2)]
```
I think this can be reduced.

Reply:
Thanks for flagging. I started digging at this, and it turns out I incorrectly took this from the EIP-7069 test. I'll push a fix

Reply:
Ah, on the number of required salt_addresses - it is actually 4 - as many as the number of times the counter may be bumped. This is related to the number of calls to the `address_subject` in `gas_test.py`. It is very ugly to couple this test with the specifics of `gas_test.py`, but I don't really have an idea on how to fix this (without making a terrible mess).